.changeset/violet-trees-play.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-shortcuts': minor
+---
+
+Fixed shortcuts initialization when using firestore
```

Comment:
Probably want a little more information as to why this is a breaking change here too now. Maybe talking about how `.get` is now a part of the public interface, and also the usage pattern may have changed if that's important too? https://github.com/backstage/backstage/pull/12571/files#diff-5869c082499ff0afcb366fb1027a189b004c0fbf4e5720c0ece9dde49f7c2b9cR42

Reply:
Updated changeset a little, please let me know if you want to add some more info